sda-commons-server-openapi/src/test/java/org/sdase/commons/server/openapi/apps/test/HouseSearchResource.java

```diff
@@ -18,13 +21,18 @@ public class HouseSearchResource {
 
   @JsonCreator
   public HouseSearchResource(
+      @JsonProperty("filters") List<Filter> filters,
```

Comment:
For completeness, you should also set `this.filters = filters` in the constructor.

Reply:
Sure, I forgot this. Done